.github/workflows/build.yml

```diff
@@ -49,6 +49,10 @@ jobs:
       - name: Build Wheels
         env:
           CIBW_ARCHS_MACOS: arm64
+          CIBW_BEFORE_BUILD: |
+            python -m pip install "conan<2.0.0"
+            conan config init --force
+            sed -i '' 's/"16.0"/"16.0", "17.0"/' ~/.conan/settings.yml
```

Comment:
The idea that we have to patch the Conan settings file like this _feels_ weird to me, but I don't know enough to say whether it's sensible or not.  Perhaps at a minimum you could add a comment explaining what it is we're doing?

Reply:
That's extremely weird to me (since is only happens for this one arm64 instance) but I think it's one of the unavoidable results of us still sticking with conan 1.x and not moving to 2 (which is not trivial; I tried). The long term solution would probably be moving to 2. I'll add a comment.

Reply:
Fwiw, I can't even tell what package's version / whatever it is that we're patching here.  Judging by the number itself, I'm _guessing_ it's LLVM or Clang?  It'd be good to comment that too.